cypress/integration/26-register_publication/543-creator_begins_wizard_registration_and_navigates_to_description_tab/registration_description.js

```diff
@@ -0,0 +1,84 @@
+import 'cypress-file-upload';
+
+const testFile = 'example.txt';
+
+// common steps
+
+Given('Creator begins registering a Registration in the Wizard', () => {
+  cy.loginCognito(USER_WITH_AUTHOR).then((idToken) => {
+    cy.wrap(idToken).as('idToken');
+    cy.setLocalStorage('i18nextLng', 'eng');
+    cy.setLocalStorage('previouslyLoggedIn', 'true');
+    cy.visit('/');
+    cy.get('[data-testid=new-publication]').click({ force: true });
+    cy.get('[data-testid=new-publication-file]').click({ force: true });
+    cy.get('input[type=file]').attachFile(testFile);
+  });
+});
+When('they navigate to the Description tab', () => {
+  cy.get('[data-testid=publication-file-start-button]').click({ force: true });
+});
+
+// Scenario: Creator begins Wizard registration and navigates to Description tab
+Then('they see the Description tab is selected', () => {
+  cy.get('[data-testid=nav-tabpanel-description][aria-selected=true]');
+});
+And('they see fields:', (dataTable) => {
+  const fieldMap = {
+    Title: 'Title',
+    Abstract: 'Abstract',
+    Description: 'Description',
+    'NPI disciplines': 'Scientific field in Norwegian publication indicator',
+    Keywords: 'Keywords',
+    'Primary language for content': 'Primary language for content',
+    'Project association': 'Project association',
+  };
+  dataTable.rawTable.forEach((value) => {
+    cy.contains(`${fieldMap[value[0]]}`);
+  });
+});
+//   | Title                        |
+//   | Abstract                     |
+//   | Description                  |
+//   | Date published               |
+//   | NPI disciplines              |
+//   | Keywords                     |
+//   | Primary language for content |
+//   | Project association          |
+And('they see the tab Reference is clickable', () => {
+  cy.get('[data-testid=nav-tabpanel-reference]').should('be.visible');
+});
+And('they see the tab Contributors is clickable', () => {
+  cy.get('[data-testid=nav-tabpanel-contributors]').should('be.visible');
+});
+And('they see the tab Files and License is clickable', () => {
+  cy.get('[data-testid=nav-tabpanel-files-and-license]').should('be.visible');
+});
+And('they see the tab Summary is clickable', () => {
+  cy.get('[data-testid=nav-tabpanel-submission]').should('be.visible');
+});
+And('they see Next is enabled', () => {
+  cy.get('[data-testid=button-next-tab]').should('be.enabled');
+});
+And('they see Save is enabled', () => {
+  cy.get('[data-testid=button-save-publication]').should('be.enabled');
+});
+
+// Scenario: Creator sees that fields are validated on Description tab
+And('they click the Save button', () => {
+  cy.get('[data-testid=publication-title-input]').focus();
+  cy.get('[data-testid=publication-title-input]').blur();
+  // TODO works in dev, not in sandbox atm
+  //   cy.get('[data-testid=button-save-publication]').click({ force: true });
+});
+Then('they can see "Required field" error messages for fields:', (dataTable) => {

```

Comment:
her er det også to ` Then` i samme `Given`

Reply:
Sammen kommentar som over, det er to scenarioer og dermed bare en `Then` pr `Given`